io/subsystem/src/main/resources/org/wildfly/extension/io/LocalDescriptions.properties

```diff
@@ -4,6 +4,8 @@ io.remove=Operation Removes subsystem
 io.worker=Defines workers
 io.worker.add=Adds new worker
 io.worker.remove=Removes worker
+io.worker.task-core-threads=Specify the starting number of threads for the worker task thread pool.\
+  If not set, it defaults to 2.
```

Comment:
Fixed in accordance with @rsvoboda's comment on https://github.com/wildfly/wildfly-core/pull/3107#discussion_r172507513.

Reply:
@rsvoboda @dmlloyd I disagree - the default-value in the r-r-d output already contains this information